src/android/Dgram.java

```diff
@@ -187,50 +75,65 @@ public boolean execute(final String action, final JSONArray data,
         return true;
     }
 
-    private void openSocket(String action, JSONArray data, CallbackContext callbackContext) throws JSONException {
-        closeSocket();
+    private void openSocket(
+        final String action, 
```

Comment:
I don't think we need action as a parameter

Reply:
putting all the parameters on a new line is me being opinionated since we don't have a linter :)

Reply:
gareth goes against nearly 20 years of prior art - okey dokey

Reply:
hah, i can be convinced to do something else.  right now it just feels arbitrary.

Reply:
it's okay, its abitrary, but writing a story like

this

might 

be  

unconventional

hehehehe

<3

Reply:
im down with whatever - just saying for your knowledge

Reply:
since feeling is first

who pays any attention

to the syntax of things

will never wholly kiss you;

wholly to be a fool

while Spring is in the world

Reply:
https://genius.com/E-e-cummings-since-feeling-is-first-annotated